Competition2017/src/competition/subsystems/feeder/commands/StepFeederPowerCommand.java

```diff
@@ -0,0 +1,37 @@
+package competition.subsystems.feeder.commands;
```

Comment:
I think we can remove the StepFeeder commands - as I realized earlier, we can just use test mode.